packages/amplify-graphql-transformer-migrator/src/schema-inspector.ts

```diff
@@ -0,0 +1,55 @@
+import { stateManager } from 'amplify-cli-core';
+import { DocumentNode } from 'graphql/language';
+import { visit } from 'graphql';
+import { collectDirectivesByTypeNames } from '@aws-amplify/graphql-transformer-core';
+import { listContainsOnlySetString } from './utils';
+import * as fs from 'fs-extra';
+
+
+export function graphQLUsingSQL(apiName: string): boolean {
+  const teamProviderInfo = stateManager.getTeamProviderInfo();
+  const env = stateManager.getLocalEnvInfo().envName;
+  if (teamProviderInfo && teamProviderInfo[env]) {
```

Comment:
You can put `teamProviderInfo[env]` in a variable since it's used below as well. And you don't need the extra `&&` check if you use optional chaining.

Reply:
I didn't know how to do optional chaining with brackets - I'll fix that